rules/mapping.smk

```diff
@@ -48,6 +48,22 @@ rule recalibrate_base_qualities:
     wrapper:
         get_wrapper_path("gatk", "baserecalibrator")
 
+rule remove_decoy:
+    input:
+        bam = "recal/{sample}-{unit}.bam",
+        decoy = config["remove_decoy"]["bed"] 
+    output: 
+        out_rm = "decoy_rm/{sample}-{unit}.removed_reads.bam",
+        out_f = "decoy_rm/{sample}-{unit}.no_decoy_reads.bam"
+    log:
+        "logs/remove_decoys/{sample}-{unit}.log"
+    threads: 8
+    resources: mem_mb=10000
+    wrapper:
+        get_wrapper_path("samtools", "remove_decoys") # or whatever 
+    # shell:
```

Comment:
Please remove comments from rules. Its ok for development, but not production code.

Reply:
sorry, forgot to remove `# or whatever` before committing, good catch! not sure I agree with having no comments at all but totally understandable here

Reply:
My bad, I meant no commented out code. Thanks for the commit!